--- ./src/Metadata.h.orig	2023-03-01 00:49:56.952593415 +0000
+++ ./src/Metadata.h	2023-03-01 00:50:11.255545771 +0000
@@ -26,6 +26,7 @@
 #include "Types.h"
 
 #include <vector>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./include/medialibrary/IMetadata.h.orig	2023-03-01 00:58:09.325953307 +0000
+++ ./include/medialibrary/IMetadata.h	2023-03-01 00:58:20.263916874 +0000
@@ -23,6 +23,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./include/medialibrary/filesystem/IFile.h.orig	2023-03-01 01:00:11.239547235 +0000
+++ ./include/medialibrary/filesystem/IFile.h	2023-03-01 01:00:27.725492319 +0000
@@ -23,6 +23,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./src/utils/File.h.orig	2023-03-01 01:04:16.556730318 +0000
+++ ./src/utils/File.h	2023-03-01 01:04:31.993679030 +0000
@@ -23,6 +23,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./src/utils/Strings.cpp.orig	2023-03-01 01:06:20.895317203 +0000
+++ ./src/utils/Strings.cpp	2023-03-01 01:06:58.120193519 +0000
@@ -29,6 +29,7 @@
 #include <algorithm>
 #include <cctype>
 #include <cassert>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./src/utils/TitleAnalyzer.h.orig	2023-03-01 01:08:07.761962128 +0000
+++ ./src/utils/TitleAnalyzer.h	2023-03-01 01:08:26.667899312 +0000
@@ -24,6 +24,7 @@
 
 #include <string>
 #include <tuple>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./src/utils/XxHasher.h.orig	2023-03-01 01:09:38.122661901 +0000
+++ ./src/utils/XxHasher.h	2023-03-01 01:09:56.757599989 +0000
@@ -24,6 +24,7 @@
 
 #include <string>
 #include <cstddef>
+#include <cstdint>
 
 namespace medialibrary
 {
--- ./src/utils/Url.cpp.orig	2023-03-01 01:11:23.788310828 +0000
+++ ./src/utils/Url.cpp	2023-03-01 01:11:55.677204876 +0000
@@ -32,6 +32,7 @@
 #include <cstring>
 #include <algorithm>
 #include <iterator>
+#include <cstdint>
 
 namespace
 {
--- ./src/Deprecated.h.orig	2023-03-01 01:13:07.136967446 +0000
+++ ./src/Deprecated.h	2023-03-01 01:13:21.637919267 +0000
@@ -28,6 +28,7 @@
  */
 
 #include <string>
+#include <cstdint>
 
 namespace medialibrary
 {
